src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -1322,6 +1323,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       UseAttributeForTargetFrameworkInfoPropertyNames="$(_UseAttributeForTargetFrameworkInfoPropertyNames)"/>
   </Target>
 
+  <Target Name="ValidateExecutableReferencesCore"
+          DependsOnTargets="_CalculateIsVSTest;ValidateExecutableReferences"
+          AfterTargets="_GetProjectReferenceTargetFrameworkProperties">
+
+  </Target>
```

Comment:
Created this extra target without condition. The reason is I want to execute `_CalculateIsVSTest` target unconditionally, because `_CalculateIsVSTest` will set the property needed for the condition in `ValidateExecutableReferences`.